src/test/java/net/snowflake/client/jdbc/ResultSetMultiTimeZoneLatestIT.java

```diff
@@ -216,7 +216,11 @@ private void helperTestUseSessionTimeZone(
     // always true since timezone_ntz doesn't add time offset
     assertEquals(true, rs.getTimestamp("COLB").toString().equals(expectedTimestamp));
     assertEquals(useSessionTimezone, rs.getTimestamp("COLC").toString().equals(expectedTimestamp));
-    // Getting timestamp from Time column will default to epoch start date
+    // Getting timestamp from Time column will default to epoch start dateString timestampStr =
+    // obj.toString();
+    //      int indexForSeparator = timestampStr.indexOf(' ');
+    //      String timezoneIndexStr = timestampStr.substring(indexForSeparator + 1);
+    //      return SFTimestamp.convertTimezoneIndexToTimeZone(Integer.parseInt(timezoneIndexStr));
```

Comment:
I don't understand this part of code. Can you explain?

Reply:
Sorry- that was just a copy/paste went wrong that I missed!! Took this back out.